src/Api/Ecommerce/Callback.php

```diff
@@ -91,7 +91,7 @@ public function call()
         }
 
         if (isset($this->postedData['require_capture'])) {
-            $response->requireCapture = (bool)$this->postedData['require_capture'];
+            $response->requireCapture = $this->postedData['require_capture'];
```

Comment:
Looks like the property might have been incorrectly hinted:
`Property Altapay\Response\CallbackResponse::$embeddedWindow (bool) does not accept string.`

Reply:
```suggestion
            $response->requireCapture = filter_var($this->postedData['require_capture'], FILTER_VALIDATE_BOOLEAN);
```